lte/gateway/python/magma/subscriberdb/main.py

```diff
@@ -137,4 +146,4 @@ def _get_s6a_manager(service, processor):
 
 
 if __name__ == "__main__":
-    main()
+    main()
```

Comment:
add newline here

Reply:
Should I add an empty line after the main() [Line: 150]?

Reply:
correct

Reply:
I've addressed the comment.